app/views/documents/_header.html.erb

```diff
@@ -19,6 +19,9 @@
 <div class="heading-extra">
   <div class="inner-heading">
     <%= render partial: 'shared/available_languages', locals: {object: document} %>
+    <% if @document.respond_to?(:logo_url) && @document.logo_url %>
```

Comment:
Given that the `logo_url` attribute accessor will be available on all editions, could this conditional lose the initial check and simply be `if @document.logo_url`?